Automattic/jetpack
# Social: Tweak copy on admin page

Small PR that improves some copy on the Jetpack Social admin page, removes the duplicate marketing section and shows the social media connections only for connected sites.

#### Changes proposed in this Pull Request:

* Change "Get Jetpack Social" to "Connect Jetpack Social"
* Don't show the marketing section and pricing section when site is connected.
* Only show the social media connections when site is connected.

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:






* Go to the Jetpack Social admin page without being connected.
* Check that no social media connections are visible.
* Connect to Jetpack.
* Go back to admin page and ensure only social media connections are shown, nothing else.

##### Screenshots


| Before|After|
|------|-----|
|<img width="1042" alt="image" src="https://user-images.githubusercontent.com/96462/165801711-0ec7ff97-1f44-4a34-9b2b-e64a9296f596.png">|<img width="1150" alt="CleanShot 2022-04-28 at 13 41 48@2x" src="https://user-images.githubusercontent.com/96462/165801811-e88d5976-e926-4953-a249-56cad4ab964f.png">|
